tests/acceptance/stepDefinitions/provisioningContext.js

```diff
@@ -7,22 +7,38 @@ const userSettings = require('../helpers/userSettings')
 
 function createDefaultUser (userId) {
   const password = userSettings.getPasswordForUser(userId)
+  return createUser(userId, password)
+}
+
+function createUser (userId, password = false, displayName = false, email = false) {
+  const body = new URLSearchParams()
+
+  if (!displayName) {
+    displayName = userSettings.getDisplayNameOfDefaultUser(userId)
```

Comment:
we might want to create a user without displayname, with that change its impossible to create a user that is in the default users table but should not have a display name